torchvision/transforms/functional_pil.py

```diff
@@ -196,15 +207,28 @@ def pad(img, padding, fill=0, padding_mode="constant"):
 
 
 @torch.jit.unused
-def crop(img: Image.Image, top: int, left: int, height: int, width: int) -> Image.Image:
+def crop(
+    img: Image.Image,
+    top: int,
+    left: int,
+    height: int,
+    width: int,
+) -> Image.Image:
+
     if not _is_pil_image(img):
         raise TypeError('img should be PIL Image. Got {}'.format(type(img)))
 
     return img.crop((left, top, left + width, top + height))
 
 
 @torch.jit.unused
-def resize(img, size, interpolation=Image.BILINEAR, max_size=None):
+def resize(
+    img: Image.Image,
+    size: Union[Sequence[int], int],
```

Comment:
The tensor API uses List instead of Sequence. Do we want them aligned?

Reply:
Yes, I think we need to have Sequence in tensor, to align. But probably JIT does not support it.

Reply:
Yes, `torch.jit` does not support `Sequence`. Since we also have a `Union` here, we can't align them anyway. Thus, I think it is ok to go with `Sequence`.